content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Istio’s new sidecar-less ‘Ambient Mode’ reaches Beta in version 1.22"
+description: The latest release from Istio brings service mesh Layer 4 & 7 features to production readiness without sidecars.
+publishdate: 2024-05-14
```

Comment:
Looks like 1.22 is aiming for today, so I think this should be the 13th as well?
```suggestion
publishdate: 2024-05-13
```